RELEASING.md

```diff
@@ -1,13 +1,48 @@
 # Releasing Ziti
 
-Release happen by merging from the release-next branch to main.
+## Release-next Pre-requisites
 
-## Release pre-requisites
+Perform these steps in PR branches based on release-next (trunk).
 
```

Comment:
It's highly informative for me, at least. Adding some tidbits of wisdom about why it's being done manually by a maintainer instead of a bot would be helpful. For example, what you're looking for when you audit the updates, things that could go wrong when updating deps, or additional tests you perform.

Reply:
Minor adjustments so it can run with fail-fast behavior in case of error.

```bash
(set -euxo pipefail
go list -m -f '{{ .Path }} {{ .Main }}' all | grep ziti | grep -v "$(go list -m)" | grep -v dilithium | cut -f 1 -d ' ' | xargs  -n1 /bin/bash -c 'echo "Checking for updates to $@";go get -u -v $@;' ''
go mod tidy
if git diff --quiet go.mod go.sum; then
  echo "no changes"   
else   
  echo "dependency updates found"      
fi

if [ -f "zititest/go.mod" ]; then
  echo "./zititest$ go mod tidy" 
  pushd zititest > /dev/null 
  go mod tidy        
  popd > /dev/null            
fi
)
```